h/services/annotation_sync.py

```diff
@@ -0,0 +1,163 @@
+from collections import defaultdict
+
+from dateutil.parser import isoparse
+
+from h.db.types import URLSafeUUID
+from h.models import Annotation
+from h.search.index import BatchIndexer
+
+
+class Result:
+    """String values for logging and metrics."""
+
+    # These are in the style of New Relic custom metric names.
+    SYNCED_MISSING = "Synced/{tag}/Missing_from_Elastic"
+    SYNCED_DIFFERENT = "Synced/{tag}/Different_in_Elastic"
+    SYNCED_FORCED = "Synced/{tag}/Forced"
+    SYNCED_TAG_TOTAL = "Synced/{tag}/Total"
+    SYNCED_TOTAL = "Synced/Total"
+    COMPLETED_UP_TO_DATE = "Completed/{tag}/Up_to_date_in_Elastic"
+    COMPLETED_DELETED = "Completed/{tag}/Deleted_from_db"
+    COMPLETED_FORCED = "Completed/{tag}/Forced"
+    COMPLETED_TAG_TOTAL = "Completed/{tag}/Total"
+    COMPLETED_TOTAL = "Completed/Total"
```

Comment:
These are some of the metrics that power the [Job Queue Dashboard](https://onenr.io/07wk7k2B0wL) in New Relic. They also get logged to Papertrail each time the sync annotations task runs.